apps/OboeTester/app/src/main/cpp/jni-bridge.cpp

```diff
@@ -213,6 +226,15 @@ Java_com_mobileer_oboetester_OboeAudioStream_getBufferSizeInFrames(
     return result;
 }
 
+JNIEXPORT void JNICALL
+Java_com_mobileer_oboetester_OboeAudioStream_setPerformanceHintEnabled(
+        JNIEnv *env, jobject, jint streamIndex, jboolean enabled) {
+    std::shared_ptr<oboe::AudioStream> oboeStream = engine.getCurrentActivity()->getStream(streamIndex);
+    if (oboeStream != nullptr) {
+        oboeStream->setPerformanceHintEnabled(enabled);
```

Comment:
If the user checks the checkbox before the stream is opened, would it be ignored? Should we either disable the checkbox before the stream is actually opened?

Reply:
Good idea. I disabled it.